### DISCUSSION TITLE
Genymotion Desktop Installer weird behavior.

### DISCUSSION THREAD

**Discussion by anonghuser** [Genymobile/feedback#54] [Genymotion Desktop] [answered]
First, when I ran the installer, it kept prompting for allowing VirtualBox to make changes 5 or 6 times. Also it prompted once for PowerShell. Why not just ask for admin access once?
And speaking of that, how can I find out what exactly did you change on my system with that Powershell? Usually app installers are expected to just change their own files in Program Files folder, if your app does something more global it would be nice to know beforehand.

Second, I thought I was installing a local android emulator, but when it started, it was asking me for an online account for your website. So, please make it clearer what that Genymotion Desktop thing is as it apparently is some partially online hosted thing I did not expect. There will probably be other confused users too.

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @anonghuser ,

> when I ran the installer, it kept prompting for allowing VirtualBox to make changes 5 or 6 times

The installer calls VirtualBox to create and activate a host-only interface - this is required for Genymotion to communicate with VirtualBox. Unfortunately, this requires several actions on Windows which requires admin access. This is a limitation of VirtualBox on Windows; it does not happen on Linux or macOS. You can disable Windows User Account Control (UAC) to skip these warnings, but this may have a negative impact your system security.

> And speaking of that, how can I find out what exactly did you change on my system with that Powershell?

If you open Windows Device Manager, you will see new "VirtualBost host-only ethernet adpater" in the Network section. You can also open VirtualBox and go to **Files > Tools > Network Manager** to see the host-only interfaces.

> Usually app installers are expected to just change their own files in Program Files folder, if your app does something more global it would be nice to know beforehand.

You're right. We will mention this in the future.

> Second, I thought I was installing a local android emulator, but when it started, it was asking me for an online account for your website.

This is already mentioned in [our on-line user guide](https://docs.genymotion.com/desktop/Get_started/Requirements/) but we will try to make it clearer in the future. Also, note that it is possible to use Genymotion Desktop off-line for some time under certain conditions, see [this article](https://genymotion.zendesk.com/hc/en-us/articles/6136960683549).

Thank you for your feedback!
Best regards.
